prql-compiler/src/lib.rs

```diff
@@ -8,25 +8,25 @@
 //!            PRQL
 //!
 //!    (parse) │ ▲
-//! pl_of_prql │ │ prql_of_pl
+//! prql_to_pl │ │ pl_to_prql
 //!            │ │
-//!            ▼ │      pl_of_json
+//!            ▼ │      json::to_pl
 //!                   ────────►
 //!           PL AST            PL JSON
 //!                   ◄────────
-//!            │        json_of_pl
+//!            │        json::from_pl
 //!            │
 //!  (resolve) │
-//!   rq_of_pl │
+//!   pl_to_rq │
 //!            │
 //!            │
-//!            ▼        rq_of_json
+//!            ▼        json::to_rq
```

Comment:
Based on the tests below, I think the labels in lines 23 and 27 should be the other way around.